packages/@coorpacademy-components/src/template/my-learning/index.js

```diff
@@ -189,6 +189,12 @@ const MyLearning = (props, context) => {
 
   const learningHoursData = useMemo(() => sumKpi('learningHours'), [sumKpi]);
 
+  const handleLearningTimeFormat = useCallback(learningTime => {
+    const hours = Math.floor(learningTime / 60);
+    const minutes = learningTime % 60;
```

Comment:
I'd introduce a `formatMinutes`, that would handle the whole logic. you dont want 1 0m, 0h 12m somethings

you could unit tests for it.